extra/release_tool.py

```diff
@@ -1200,6 +1197,13 @@ def main():
                         + "pull requests triggering a failure)")
     args = parser.parse_args()
 
+    if not os.getenv("JENKINS_USER") or not os.getenv("JENKINS_PASSWORD"):
+        logging.warn("WARNING: JENKINS_USER and JENKINS_PASSWORD env. variables not set")
+
+    global JENKINS_USER, JENKINS_PASSWORD
+    JENKINS_USER = os.getenv("JENKINS_USER")
+    JENKINS_PASSWORD = os.getenv("JENKINS_PASSWORD")
+
```

Comment:
I think this section can be moved into `do_release`, since it won't be used in any other section, and the tool is used a lot outside of that as well.